app/models/routing/routing_tag.rb

```diff
@@ -69,7 +74,7 @@ def prevent_destroy_if_have_assosiations
       errors.add(:base, "Can't be deleted because linked to Rate Management Project(s) ##{project_ids.join(', #')}")
     end
 
-    pricelist_ids = rate_management_pricelist_items.pluck(Arel.sql('DISTINCT(pricelist_id)'))
+    pricelist_ids = active_rate_management_pricelist_items.pluck(Arel.sql('DISTINCT(pricelist_id)'))
```

Comment:
here it will be nice to leave a comment on why we validate only active pricelist items  
```ruby
# we allow keeping deleted routing tag IDs in applied rate management pricelist items
# because it does not break anything and such records are used only as a temporary history
```